### PR TITLE
Fix Entity instance in collapse mode, fix and improve tooltip 

### PR DESCRIPTION
**Pull request by larisaVasile**

### CODE REVIEW

File: core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts [hunk 1]
```diff
@@ -480,8 +480,14 @@ export class MxGraphVisitorHelper {
...
       const [elementVersionedNamespace] = modelElement.aspectModelUrn.split('#');
 
+      let metaModelVersion =  modelElement.metaModelVersion;
+      if(modelElement instanceof DefaultEntityInstance){
+        metaModelVersion = modelElement.entity.metaModelVersion;
+      }
+
       return {
-        version: modelElement.metaModelVersion,
+        version: this.extractText(rdfModel.absoluteAspectModelFileName,':',':'),
```
Comment (michelu89) [inline core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts:489]: Have a look on rdf-model-utils.ts, there are some static methods that you can use to extract the version.

File: core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts [hunk 2]
```diff
@@ -494,4 +500,16 @@ export class MxGraphVisitorHelper {
       return null;
     }
   }
+
+  static extractText(inputString: string, startChar: string, endChar: string) {
```
Comment (michelu89) [inline core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts:504]: Use rdf-mode-utils to do this.

**Review by michelu89 — CHANGES_REQUESTED** [review]

File: core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts (hunk 1 above)
Comment (larisaVasile) [reply core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts:489]: done

File: core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts (hunk 2 above)
Comment (larisaVasile) [reply core/libs/mx-graph/src/lib/helpers/mx-graph-visitor-helper.ts:504]: done

**Review by michelu89 — APPROVED** [review]

**Pull request merged**